peps/pep-0736.rst

```diff
@@ -238,6 +230,44 @@ for the ``f(x=)`` form for the following reasons:
 * `A poll of Python developers <https://discuss.python.org/t/syntactic-sugar-to-encourage-use-of-named-arguments/36217/130>`__
   indicates that this is the most popular syntax among those proposed.
 
+How to Teach This
+=================
+
+Keen Python developers will likely hear about this feature through typical
+information channels, such as newsboards, social media, mailing lists, online
+forums, or word of mouth. Many more will encounter this feature while reading
+code and noting the omission of the value in a keyword argument at invocation,
+violating their expectations. We should ensure such developers have easy access
+to documentation that explains the semantics of this feature and that this
+documentation is easy to find when searching. For example, the
+`Python Glossary <https://docs.python.org/3/glossary.html#term-argument>`__ and
+`Tutorial <https://docs.python.org/3/tutorial/controlflow.html#keyword-arguments>`__
+may be updated accordingly and reasonable keywords may be used to help with
+search discoverability.
+`A StackOverflow question <https://stackoverflow.blog/2011/07/01/its-ok-to-ask-and-answer-your-own-questions/>`__
+could be written to help explain this feature to those searching for an
+explanation.
+
+To ease the communication of and search for this feature, it may also be
```

Comment:
Let's make this the first paragraph of the How to Teach This section. I like how you called out giving a name to reference.